project-base/storefront/components/Basic/Overlay/Overlay.tsx

```diff
@@ -8,34 +10,29 @@ type OverlayProps = {
 };
 
 export const Overlay: FC<OverlayProps> = ({ onClick, isActive, isHiddenOnDesktop, children }) => {
-    useEffect(() => {
-        if (isActive) {
-            document.body.style.overflow = 'hidden';
-        } else {
-            document.body.style.overflow = 'auto';
-        }
-
-        return () => {
-            document.body.style.overflow = 'auto';
-        };
-    }, [isActive]);
```

Comment:
Without this, we can now scroll the content behind the popup. Is that desired?

Reply:
Yes, I replaced it with the react-remove-scroll library because it prevents the layout from shifting when the popup opens